### PR TITLE
Add request authorisation to dropbox API

### PR DESCRIPTION
**Pull request by ShraddhaAg**
Issue #8 
Authorisation URL added.

### CODE REVIEW

File: js/paper.js [hunk 1]
```diff
@@ -39,7 +39,9 @@ chrome.contextMenus.onClicked.addListener(function(info, tab) {
 });
 
 function authorise() {
-  //TODO: create new tab to request authorisation (use "https://wdlsvnit.github.io/paper-extension/" as redirect url)
```
Comment (AdityaHirapara) [inline js/paper.js]: Remove this comment

File: js/paper.js [hunk 2]
```diff
@@ -39,7 +39,9 @@ chrome.contextMenus.onClicked.addListener(function(info, tab) {
...
+    dropboxURL = "https://www.dropbox.com/oauth2/authorize?response_type=code&client_id=4xu6bbav7553wwg&redirect_uri=https://wdlsvnit.github.io/paper-extension/"
```
Comment (AdityaHirapara) [inline js/paper.js]: Two space indentation
Declare variable using var 
use clientId variable(which is declared at top) to create URL and don't push your client id.

File: js/paper.js [hunk 3]
```diff
@@ -39,7 +39,9 @@ chrome.contextMenus.onClicked.addListener(function(info, tab) {
...
+  chrome.tabs.create({ url: dropboxURL });
+
```
Comment (AdityaHirapara) [inline js/paper.js]: Remove this blank line

**Comment by AdityaHirapara** [comment]
Good work!

**Comment by ajayns** [comment]
@ShraddhaAg Create a new branch each time you open a PR. The master branch should be kept up to date with the master of the original repo you forked so that if you're working on multiple issues simultaneous you can create different branches off master solving the issues.

**Pull request merged**